docs/install-crossplane.md

```diff
@@ -269,6 +269,7 @@ The following tables lists the configurable parameters of the Crossplane chart a
 | `clusterStacks.rook.version`     | Rook stack version to deploy                                    | `<latest released version>`   
 | `personas.deploy`                | Install roles and bindings for Crossplane user personas         | `true`     
 | `templateStacks.enabled`         | Enable experimental template stacks support                     | `false`     
+| `templateStacks.controllerImage` | Template Stack controller image                                 | `bymafmaf/templating-controller:0.0.1`
```

Comment:
Would the intended default be `crossplane/resourcepacks:master` or something at the time that this is merged? Making a note in case that's intended, so it's easier to remember